Update license to v1.2 (#1711)
slint-ui/slint: README.md

[![REUSE status](https://api.reuse.software/badge/github.com/slint-ui/slint)](https://api.reuse.software/info/github.com/slint-ui/slint)
[![Discussions](https://img.shields.io/github/discussions/slint-ui/slint)](https://github.com/slint-ui/slint/discussions)

> Slint is a toolkit to efficiently design, develop and deploy responsive apps on any screen. Frontends built with Slint can be easily integrated with backends written in different programming languages such as C/C++, Rust, and JavaScript. Slint is developed by SixtyFPS GmbH together with the community. We invite you to use Slint and be part of its community.

Reviewer: Interesting introduction of frontend and backend here.

Author: Open to changes to wording :)

Reviewer: Re-reading this, I think the previously sentence was simpler and easier to read. Introducing terms such as "backend" and "frontend" seems tempting from a technical perspective, but it also creates ambiguity for people coming from the web for example. Does this mean that the backend runs on a server and Slint UI runs elsewhere? (traditionally there's a host boundary between those).

Since this is the very first sentence of the README, simplicity and clarify wins over the "feature" that we support a frontend and backend separation. If anything, I think that could be a bullet point under "features" somewhere.

I suggest to go back to the old wording.

Reviewer 2: Maybe Design vs. Logic? 

```suggestion
Slint is a toolkit to efficiently design, develop and deploy responsive apps on any screen. User interface design built with Slint can be easily integrated with logic written in different programming languages such as C++, Rust, and JavaScript. Slint is developed by SixtyFPS GmbH together with the community. We invite you to use Slint and be part of its community.
```

I'm also unsure if we should mention SixtyFPS GmbH so early in the introduction.

## Design Goals

We use the following design goals as a guide when developing Slint

>  - **Scalable**: Run on any screen connected to a device, from desktop computers to low end embedded systems.

Reviewer: ```suggestion
 - **Scalable**: Run on any screen connected to a device, from desktop computers to low-end embedded systems.
```

 - **Lightweight**: Fit into a few hundred kilobytes of RAM and require little processing power.
 - **Intuitive**: Both developers and designers should feel productive and enjoy the design and development process.
   The APIs should be consistent and easy to use, no matter the target language.
 - **Native**: Slint apps should match the users' expectations of a native application. Various target platforms such as 
   embedded devices, desktops, mobile and web should be supported so that both the user and the developer feel comfortable on
   their platform of choice.

> ## Get Started

Reviewer 2: I wonder if we can't just leave out this paragraph.  All the information is somewhere else in the readme

Reviewer: Agreed.

Author: Removed

Reviewer 2: Eh, I was talking about the Quick Start paragraph,  not the design goals.  @tronical : What paragraph where you talking about?

I think the design goal are good to keep. Or do you think it is not relevant anymore on this page?

Reviewer: Yes, I was also talking about the "Quick Start" paragraph. The design goals are good IMO to have on this landing page.

Author: removed Quick Start and retained Design Goals

Check the [online demos](#demos-running-in-webassembly-simulation) or
[build on your own system](docs/building.md). Also check out the [documentation](#documentation).

## Current Status

Slint is in active development. The state of the toolkit for each platform is as follows:

 - **Embedded**: *Ready!* Slint is already being used in production projects by customers 
   (running on an Arm processor with Linux).
 - **Microcontrollers** (MCU): *Ready*. Slint apps can run on MCUs with less than 300K of RAM.
   [Video of Slint on STM32](https://www.youtube.com/watch?v=NNNOJJsOAis) –
   [Video of Slint on RP2040](https://www.youtube.com/watch?v=dkBwNocItGs) – 
   [docs](https://slint-ui.com/docs/rust/slint/docs/mcu/index.html)
 - **Desktop**: *In Progress*. While Slint is usable on Windows, Linux and Mac, we are working on
    improving the platform support in subsequent releases to reach the level of integration that

>     have aimed for.

Reviewer 3: "... that *we* have aimed for." ?

Or maybe just

> While Slint is usable on Windows, Linux and Mac, we are working on improving the platform support in subsequent releases.

Author: Updated

 - **Mobile** (Android/iOS): *Todo*. We haven't started supporting mobile platforms yet, but it
   is our intention to do so in the near future.
 - **Web**: *In Progress*. Slint apps can be compiled to WebAssembly and can run in a web browser.
   As there are many other web frameworks, the web platform is not one of our primary target platforms.
   The web support is currently limited to demo purposes.

### Accessibility

>  Slint supports keyboard based navigation of many widgets, and user interfaces are scalable. 

Reviewer 3: IIRC the actual markup standard requires a blank line after a headline (even though none of the renderers seems to care;-).

Author: Done

 - **[Project Trains Launcher](https://github.com/Project-Trains/launcher)**: Cross-platform game launcher made for Project Trains simulator.
 - **[Mastermind](https://github.com/ElevenJune/mastermind_Rust)**: Mastermind game coded in Rust.

## License

Slint can be used under either a [commercial license](./LICENSES/LicenseRef-Slint-commercial.md) or [GNU GPLv3](./LICENSES/GPL-3.0-only.txt), at your choice.

> The commercial license is provided for free if you help us promote Slint: Check out our

Reviewer 3: is provided => can be provided ?

Author: Done

[ambassador license](https://slint-ui.com/ambassador-program.html).

See also the [Licensing FAQ](FAQ.md#licensing)

## Contributions

We welcome your contributions: in the form of code, bug reports or feedback.

 * If you see an [RFC tag](https://github.com/slint-ui/slint/labels/rfc) on an issue, feel free to
   chime in.
 * For contribution guidelines see [CONTRIBUTING.md](CONTRIBUTING.md). The dual-licensing of Slint requires the
   contributor to accept a CLA.

## Frequently Asked Questions

Please see our separate [FAQ](FAQ.md).

## About us

We are passionate about software - API design, cross-platform software development and user interface components.

> Our aim to make developing user interfaces fun for everyone: from JavaScript, C++, or Rust developers all the 

Reviewer: ```suggestion
Our aim is to make developing user interfaces fun for everyone: from JavaScript, C++, or Rust developers all the 
```
